examples/failing/TypedBinders.purs

```diff
@@ -0,0 +1,49 @@
+module Main where
```

Comment:
I am using this example only because it's referenced in the original issue; a simpler one would be great!